EIPS/eip-draft_transaction_representation.md

```diff
@@ -0,0 +1,67 @@
+---
+eip: 7023
+title: Automatic Transaction Representation
+description: Living guidelines for automatically generating a human-readable representation of a transaction and its corresponding EIP-712 signature.
```

Comment:
Why living?  This seems like something that should be a solidified standard so people can implement it reliably.

Reply:
[![Auto Review Bot](https://github.com/ethereum/EIPs/actions/workflows/auto-review-bot.yml/badge.svg)](https://github.com/ethereum/EIPs/actions/workflows/auto-review-bot.yml)